src/Faker/Provider/Internet.php

```diff
@@ -331,4 +331,128 @@ public static function macAddress()
 
         return $mac;
     }
+
+    /**
+     *
+     */
+    const SMALLEST_HTML_LENGTH = 4; // <br>
+
+    /**
+     * @param string $tag
+     * @param string $content
+     * @param string $attributes as a string
+     *
+     * @return string
+     */
+    protected static function buildHtmlTag($tag, $content = '')
+    {
+        return sprintf('<%s>%s</%s>', $tag, $content, $tag);
+    }
+
+    /**
+     * @return array of distinct block-level and inline-level elements names
+     */
+    protected static function getHtmlTags()
+    {
+        return array(
```

Comment:
You should move that to a static property to let devs override it in subclasses.